forge-gui/res/cardsfolder/upcoming/dredging_claw.txt

```diff
@@ -0,0 +1,10 @@
+Name:Dredging Claw
+ManaCost:2
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 2 | AddKeyword$ Menace | Description$ Equipped creature gets +1/+0 and has menace.
```

Comment:
`AddPower$ 1`

Reply:
Looks like you resolved this too quickly

Reply:
whoops forgot to push